components/SingleArticle.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { BodyHeader } from "./BodyHeader";
+import { getArticleById } from "../utils/api";
+import { useParams } from "react-router-dom";
+
+export const SingleArticle = () => {
+  const [article, setArticle] = useState([]);
+
```

Comment:
we could do with some state in order to allow feedback for when the page is loading